### ISSUE TITLE
Proposal: Update that 

### ISSUE THREAD

**Issue by frank-dspeed** [Macrofig/docker-donejs#1]
We should refactor and update that and also make it default used by vagrant-donejs wich is currently named donejs-vagrant 

Related:
- https://github.com/donejs/donejs-vagrant/issues/3 - lets wait with the update till the old repo is renamed

**Comment by pYr0x** [comment]
do you want to refactor this?

**Comment by frank-dspeed** [comment]
@pYr0x yes i am ready to do that and also publish it to the docker offical images
